modules/config.js

```diff
@@ -1,5 +1,5 @@
+const fsPromises = require('./fs_extensions');
```

Comment:
This isn't a huge problem, but it's generally safer to leave the original module as-is and just import the extra functions you need.
```js
const fsPromises = require('fs').promises;
const fsExtensions = require('./fs_extensions');
```

This is pretty minor, since the only real benefit is that you don't have to worry about a future node version introducing a function that clashes with the ones you defined. (Imagine for example that node includes a `fsPromises.fileExists` function in a future release that works slightly different than the one you have).

But still, I don't see why not doing it that way 🤷

Reply:
ah i see
but then the real problem lies in fs_extensions.js:
```
module.exports = Object.assign(fsPromises, extensions); 
```
here I'm already "overriding" the fsPromise functions

maybe something like this makes more sense?
```
var exp = Object.assign({}, fsPromises); 
module.exports = Object.assign(exp, extensions); 
```
starting with an empty object I add the original fsPromise functions and then the extensions
that way 

- i keep the original fsPromise functions intact 
if needed you can "require('fs').promises" to get the original

- and the extension export object has both original fsPromise and extension functions 
(extensions "overriding" eventually already present fsPromise functions) 
so you usually are ok with just "require('./fs_extensions')" to get both